src/index.js

```diff
@@ -1,36 +1,69 @@
 import './style.css';
 import './imgs/bg1.png';
-
-const arrTodo = [
-  {
-    description: 'do homework',
-    completed: true,
-    index: 3,
-  },
-  {
-    description: 'clean house',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'play soccer',
-    completed: false,
-    index: 2,
-  },
-];
+import Task from '../modules/ntask.js';
+import addtask from '../modules/addtask.js';
+import removetask from '../modules/removetask.js';
+import edittask from '../modules/edittask.js';
 
 const todoList = document.getElementById('todoList');
 
-arrTodo.sort((a, b) => a.index - b.index);
-
-arrTodo.forEach((item) => {
-  todoList.innerHTML += `
-    <div class="todoFlex">
-      <div class="todoDiv">
-        <input type="checkbox" name="check" id="" ${item.completed ? 'checked' : ''} >
-        <p>${item.description}</p>
-      </div>
-      <i class="bi bi-three-dots-vertical"></i>
-    </div>
-  `;
+// displaying tasks on window loading
+window.addEventListener('DOMContentLoaded', () => {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+
+  if (tasks !== null) {
+    tasks.sort((a, b) => a.index - b.index);
+
+    tasks.forEach((item) => {
+      todoList.innerHTML += `
+        <div class="todoFlex">
+          <div class="todoDiv">
+            <input type="checkbox" name="${item.id}" class="checkbox" ${item.completed ? 'checked' : ''} >
+            <input class="todoP" name="${item.id}" value="${item.description}" >
+          </div>
+          <i class="bi bi-three-dots-vertical dots"></i>
+        </div>
+      `;
+    });
+  }
+});
+
+// formInput + enter
+const formInput = document.querySelector('.formInput');
+const enter = document.getElementById('enter');
+
+formInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    e.preventDefault();
+  }
+});
+
+let ind;
```

Comment:
- [ ] KIndly make sure you always use a variable name that is easy to understand so that your code can be easy to maintain 👍🏽